### PR TITLE
Fix/Server Port

### PR DESCRIPTION
**Pull request by Abelitee**
### What does this PR do?
This updates the server port

### Description of the task to be completed
the port was changed to a functioning one compared to the previous one.

### Acceptance Criteria

- As a user
- WHEN \*\*\*
- THEN \*\*\*

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import app from './app';
 
 // listen for requests
-const PORT = process.env.PORT || 4000;
+const PORT = process.env.PORT || 3000;
```
Comment (alob-mtc) [inline src/index.js:4]: Please add a comment here thanks

**Comment by alob-mtc** [comment]
Hello

File: src/index.js (hunk 1 above)
Comment (alob-mtc) [reply src/index.js:4]: @Abelitee

Comment (Abelitee) [reply src/index.js:4]: it will be added in the next commit

**Pull request merged**